### PR TITLE
Add Ricoh Theta

### PR DESCRIPTION
**Pull request by rioriost**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

After making all changes to a cask, verify:

- [ ] `brew cask audit --download {{cask_file}}` is error-free.
- [ ] `brew cask style --fix {{cask_file}}` reports no offenses.
- [ ] There are no [open pull requests](https://github.com/Homebrew/homebrew-cask/pulls) for the same update.
- [ ] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#stable-versions) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, **if adding a new cask**:

- [ ] Named the cask according to the [token reference](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [ ] `brew cask install {{cask_file}}` worked successfully.
- [ ] `brew cask uninstall {{cask_file}}` worked successfully.
- [ ] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [ ] Checked the cask is submitted to [the correct repo](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask).

### CODE REVIEW

File: Casks/ricoh-theta.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+cask 'ricoh-theta' do
+  version '3.13.3'
+  sha256 '6903ff2fa7c38a59c0d5562c51dade624d3e8de6acf908f9bf1df31913e9accf'
+
+  url 'https://support.theta360.com/intl/download/pcapp/macosx'
```
Comment (miccal) [inline Casks/ricoh-theta.rb:5]: Please change this to:
```
https://theta360-statics.s3.amazonaws.com/app/viewer/RICOH%20THETA.dmg
```
and please add the following `appcast`:
```
appcast 'https://support.theta360.com/en/download/pcmac/'
```

File: Casks/ricoh-theta.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+  name 'RICOH THETA'
+  homepage 'https://theta360.com/ja/'
```
Comment (miccal) [inline Casks/ricoh-theta.rb:7]: Please change this to:
```
homepage 'https://theta360.com/'
```

**Comment by miccal** [comment]
Thanks @rioriost, but the `appcast` should be between the `url` and `name` stanzas.

File: Casks/ricoh-theta.rb [hunk 3]
```diff
@@ -0,0 +1,12 @@
+cask 'ricoh-theta' do
+  version '3.13.3'
+  sha256 '6903ff2fa7c38a59c0d5562c51dade624d3e8de6acf908f9bf1df31913e9accf'
+
+  # theta360-statics.s3.amazonaws.com was verified as official when first introduced to the cask
+  url 'https://theta360-statics.s3.amazonaws.com/app/viewer/RICOH%20THETA.dmg'
+  appcast 'https://support.theta360.com/en/download/pcmac/'
+  name 'RICOH THETA'
+  homepage 'https://theta360.com/'
```
Comment (vitorgalvao) [inline Casks/ricoh-theta.rb:9]: Make it something more specific to the app.

**Comment by vitorgalvao** [comment]
Needs to be submitted to [homebrew/cask-drivers](https://github.com/Homebrew/homebrew-cask-drivers). We [define drivers](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) as:

> Software to make a hardware peripheral recognisable and usable by the system. If the software is useless without the peripheral, it’s considered a driver.

Note that repo has [special naming rules](https://github.com/Homebrew/homebrew-cask-drivers/blob/master/CONTRIBUTING.md#naming-driver-casks).

**Pull request closed without merging**